website/content/docs/security/acl/acl-endpoints-by-resource.mdx

```diff
@@ -0,0 +1,222 @@
+---
+layout: docs
+page_title: Consul API Endpoint Access List
+description: |-
+  Consul API Access list based on the action and access level
+---
+
+# API Endpoint Access List
+
+Below are lists of API endpoints organized by the resource and access level that controls. Write access also provides read access.
+
```

Comment:
@markan : we should add a section for which endpoints are accessible without need for any ACL permissions